pycbc/inference/sampler.py

```diff
@@ -181,8 +276,14 @@ def run(self, niterations, **kwargs):
             The list of log proposal densities for the walkers at positions p,
             with shape (nwalkers, ndim).
         """
-        return self._sampler.run_mcmc(niterations, **kwargs)
+        try:
+            p0 = kwargs.pop('p0')
+        except KeyError:
+            raise ValueError("must provide a p0 in the keyword arguments")
+        return self._sampler.run_mcmc(p0, niterations, **kwargs)
+
 
 samplers = {
     "kombine" : KombineSampler,
```

Comment:
Both kombine and emcee are ensemble samplers. I think should call them either ``KombineEnsembleSampler`` and ``EmceeEnsembleSampler`` or ``KombineSampler`` or ``EmceeSampler``.